docs/proposals/scheduling/20251209-support-job-queuing-and-managment.md

```diff
@@ -0,0 +1,303 @@
+# Proposal: Support Job Queueing and Management in Koordinator
+
+## Motivation
+
+In multi-tenant or batch-heavy Kubernetes environments, efficient job queueing and prioritization is critical for resource fairness and system stability. While Koordinator already provides advanced scheduling policies and resource management capabilities, it lacks native support for **structured job queueing** and **fine-grained priority management**. This proposal aims to integrate a robust queueing system inspired by [kube-queue](https://github.com/kubernetes-sigs/kube-queue) to address the following needs:
+
+*   **Orderly job execution**: Ensure jobs are processed in a predictable sequence based on user-defined rules.
+    
+*   **Priority-based scheduling**: Prevent low-priority workloads from starving high-priority tasks.
+    
+*   **Job lifecycle management:** Manage job's lifecycle.
+    
+*   **Job-level resource checking:** Support check resource in queue to avoid too many pods in scheduler.
+    
+
+## User Story
+
+*   My company provide AI model training platform for users. Users submit training jobs to the platform and pay for the resources. There will be a lot of pods in the cluster and cause some problems like system components OOM. Too many unschedulable pods will also cost many schedule cycles which leads to the high schedule latency. So I need to use a queue component to reduce the number of pods in the cluster. 
```

Comment:
It is recommended to provide details from both the cluster-admin and user perspectives regarding which components need to be installed, what YAML configurations are required globally, what YAML configurations are required at the user level, and what YAML configurations are required at the job level.